src/workspace.ml

```diff
@@ -40,60 +40,66 @@ module Context = struct
         name)
   end
 
-  module Opam = struct
+  module Base = struct
```

Comment:
`Common` would look slightly clearer to me

Reply:
Renamed